cobbler/web/settings.py

```diff
@@ -20,7 +20,7 @@
 DATABASE_PORT = ''
 
 # Force Django to use the systems timezone
-TIME_ZONE = None
+TIME_ZONE = False
```

Comment:
Use this instead. Otherwise Django in combination with `python3-tz` will not show proper error-pages.

```
# Force Django to use a specific timezone and TZ
TIME_ZONE = 'UTC'
USE_TZ = True
```

`UTC` can also be set to localtime for example `Europe/Amsterdam`.

Reply:
Also mentioned in issue #2265.

Reply:
Fixed through: 7d90c8d